src/workerd/api/tests/commonjs-module-test.js

```diff
@@ -0,0 +1,10 @@
+import { foo, bar, default as baz } from 'foo';
```

Comment:
could you please also test star imports?

```
import * as foo from 'foo';
```

and ensure that foo is defined with they foo, bar, baz keys?

Reply:
Done.